src/ui/WidgetEbt.vue

```diff
@@ -1,14 +1,12 @@
 <template>
-  <BaseWidget v-if="connected" :link="hasEbt ? '/ebt' : ''" :status="'neutral'">
+  <BaseWidget v-if="connected" :link="hasEbt ? '/ebt' : ''" :status="numPeers ? 'good' : 'neutral'">
     <div>
-      <div>
-        Ebt status:
-      </div>
+      <div></div>
       <div v-if="hasEbt">
-        The client has ebt support. Click for more info.
+        The client can gossip, and currently sees {{ numPeers }} peers
```

Comment:
ebt is a newer more efficient way of gossiping, it's not required for it. E.g. patchwork didn't enable ebt until the [most recent version](https://github.com/ssbc/patchwork/releases/tag/v3.17.1) but was gossiping before then.

Reply:
Also `sees {{ numPeers }} peers` is incorrect, we might see more clients. `numPeers` is the number of clients we're connected to that have ebt support (I believe).